daniel.py

```diff
@@ -5,15 +5,17 @@
 wiek = input("Podaj ile masz lat: ")
 wiek = int(wiek)
 uro = input("Czy miałeś/aś już w tym roku urodziny? (tak/nie): ")
-if uro == ("tak"):
-    kiedy_urodziny = x.year - wiek + 100
-    kiedy_urodziny = str(kiedy_urodziny)
-    print("Super sprawa " + imię + " w " + kiedy_urodziny + " roku będziesz miał/a 100 lat")
-else:
-    kiedy_urodziny = x.year - wiek + 99
-    kiedy_urodziny = str(kiedy_urodziny)
-    print("Super sprawa " + imię + " w " + kiedy_urodziny + " roku będziesz miał/a 100 lat")
-print("A teraz mój system obliczeniowy sprawdzi czy podana przez ciebie liczba jest parzysta")
+urodzinytak = x.year - wiek + 100
```

Comment:
It's a weird combination of polish and English names. It's good to be consistent and preferably stick to English.